### PR TITLE
Bugfix: unmarshal issue with byte slice in packetIn2 message

### PR DESCRIPTION
**Pull request by wenyingd**
Use "copy" function to set the byte slice fields in a message.

### CODE REVIEW

File: ofctrl/ofSwitch.go [hunk 1]
```diff
@@ -582,11 +582,12 @@ func (self *OFSwitch) ResumePacket(pktIn *PacketIn) error {
 	}
 	matchProp.Length = matchProp.Len()
 	continueProp := &openflow15.PacketIn2PropContinuation{
-		Continuation: pktIn.Continuation,
+		Continuation: make([]byte, len(pktIn.Continuation)),
 		PropHeader: &openflow15.PropHeader{
 			Type: openflow15.NXPINT_CONTINUATION,
 		},
 	}
+	copy(continueProp.Continuation[0:], pktIn.Continuation)
```
Comment (tnqn) [inline ofctrl/ofSwitch.go:590]: why `0:` is required?

File: ofctrl/ofctrl.go [hunk 2]
```diff
@@ -62,9 +62,11 @@ func parsePacktInFromNXPacketIn2(pktIn2 *openflow15.PacketIn2) *PacketIn {
 				Fields: v.Fields,
 			}
 		case *openflow15.PacketIn2PropUserdata:
-			userData = v.Userdata
+			userData = make([]byte, len(v.Userdata))
+			copy(userData[0:], v.Userdata)
```
Comment (tnqn) [inline ofctrl/ofctrl.go:66]: ditto

File: ofctrl/ofctrl.go [hunk 3]
```diff
@@ -62,9 +62,11 @@ func parsePacktInFromNXPacketIn2(pktIn2 *openflow15.PacketIn2) *PacketIn {
...
 		case *openflow15.PacketIn2PropContinuation:
-			continuation = v.Continuation
+			continuation = make([]byte, len(v.Continuation))
+			copy(continuation[0:], v.Continuation)
```
Comment (tnqn) [inline ofctrl/ofctrl.go:69]: ditto

File: ofctrl/ofctrl.go (hunk 2 above)
Comment (wenyingd) [reply ofctrl/ofctrl.go]: removed.

File: ofctrl/ofctrl.go (hunk 3 above)
Comment (wenyingd) [reply ofctrl/ofctrl.go]: done.

**Review by tnqn — DISMISSED** [review]
Do you plan to add unit test here or another place?

**Comment by wenyingd** [comment]
Unit test is added in libOpenflow

**Comment by wenyingd** [comment]
Updated `go.mod` and `go.sum` to consume latest changes in libOpenflow. @tnqn could you help take a look again?

**Review by tnqn — APPROVED** [review]

**Pull request merged**